controllers/numaresourcesoperator_controller.go

```diff
@@ -222,6 +222,10 @@ func (r *NUMAResourcesOperatorReconciler) reconcileResource(ctx context.Context,
 		}
 	}
 
+	if instance.Spec.RTESync == "disabled" {
```

Comment:
this needs a log messages, perhaps at V >= 4

Reply:
done, thanks